cannon/mipsevm/exec/mips_instructions.go

```diff
@@ -479,10 +479,7 @@ func ExecuteMipsInstruction(insn uint32, opcode uint32, fun uint32, rs, rt, mem
 			return mem
 		case 0x3F: // sd
 			assertMips64(insn)
-			sl := (rs & 0x7) << 3
-			val := rt << sl
-			mask := ^Word(0) << sl
-			return (mem & ^mask) | val
+			return rt
```

Comment:
This is simplified given that `sd` assumes natural-alignment of the entire doubleword. So there's no need to adjust the value to align in a memory offset.